os/include/tinyara/log_dump/log_dump.h

```diff
@@ -41,8 +41,9 @@
  * Public Functions
  ********************************************************************************/
 
-#define OPEN_LOGDUMP()			open("/proc/logsave", O_RDWR)
-#define START_LOGDUMP_SAVE(fd)		write(fd, LOGDUMP_SAVE_START, strlen(LOGDUMP_SAVE_START) + 1)
-#define STOP_LOGDUMP_SAVE(fd)		write(fd, LOGDUMP_SAVE_STOP, strlen(LOGDUMP_SAVE_STOP) + 1)
-#define READ_LOGDUMP(fd, buf, bufsize)	read(fd, buf, bufsize)
-#define CLOSE_LOGDUMP(fd)		close(fd)
+#define OPEN_LOGDUMP()				open("/proc/logsave", O_RDWR)
+#define START_LOGDUMP_SAVE(fd)			write(fd, LOGDUMP_SAVE_START, strlen(LOGDUMP_SAVE_START) + 1)
+#define STOP_LOGDUMP_SAVE(fd)			write(fd, LOGDUMP_SAVE_STOP, strlen(LOGDUMP_SAVE_STOP) + 1)
+#define SET_LOGDUMP_FETCH_BLOCKS(fd, num)	write(fd, num, strlen(num) + 1)
```

Comment:
1. To say what it is clearly, how about `GET_LOGDUMP_FROM_LAST(fd, num)` or something else?
2. How does an app know how many nodes are? I mean when a developer makes codes with this, how does he know the number of entire saved nodes?

Reply:
1. Done, I have changed the macro to GET_LOGDUMP_FROM_LAST
2. app need not know how many nodes are present. Let's say we have n nodes of log dump, if the app tries to do "log_dump [n+k]" (trying to fetch more than n nodes from the end) then we will provide the entire contents.